README.md

```diff
@@ -30,16 +30,50 @@ month: 2
 year: 2023
 ---
 
-your markdown content
+# This is a heading style one
+
+**This is a featured Paragraph it cannot contain bold text but can contain [links](https://www.unitary.fund).**
+
+More text
+
+::image[d394zg4utwqxynenamqf]
+
+## This is a heading in heading style two
+
+This is a standard paragraph with a [LINK](https://www.google.com). **This is bold paragraph text**
+
+- this is a list item
+- this is **bold** list item
+- this is a long list item which has lots of words in it and will probably break onto a second or even a third line so we can see how that looks at differnt sizes adn make sure it looks as it is intended too
+- this is a list item with a [LINK](https://www.google.com) or [two)](https://www.google.com) added in it. 
+
+### This is a heading in heading style three introducing a video
+
+https://www.youtube.com/watch?v=dB_3R84ewig
+
+#### This is a heading in heading style four
+
+::gist[6cc14d37209de07abd707804f1b0219e]
+
+With slashes
+
+::gist[matt-lourens/6cc14d37209de07abd707804f1b0219e/]
+
+::gist[/matt-lourens//6cc14d37209de07abd707804f1b0219e/]
```

Comment:
I'd say we suggest a single one of these. Also it's unclear where `matt-lourens` is coming from . When uploading an image as described below, there is no user identifier that is embedded in the code. This either needs explanation, or removal.

Reply:
Good point. They seem to come from https://gist.github.com/matt-lourens/ and then the second is https://gist.github.com/matt-lourens/6cc14d37209de07abd707804f1b0219e.